sim/target-agnostic.mk

```diff
@@ -43,6 +50,11 @@ midas_cc = $(shell find $(simif_dir) -name "*.cc")
 common_cxx_flags := $(TARGET_CXX_FLAGS) -Wno-unused-variable
 common_ld_flags := $(TARGET_LD_FLAGS) -lrt
 
+# Simulation memory map emitted by the MIDAS compiler
+header := $(GENERATED_DIR)/$(BASE_FILE_NAME).const.h
+# The midas-generated simulator RTL which will be baked into the FPGA shell project
+simulator_verilog := $(GENERATED_DIR)/$(BASE_FILE_NAME).sv
```

Comment:
I renamed HEADER -> header, VERILOG -> simulator_verilog, to try to maintain that uppercase variables are overridable, whilst lowercase ones are not. I prepended simulator_ to avoid a confusion collision with the phony target.